// Basic messages that are used by physics objects
//
// Created by Samvel Khalatyan, Apr 19, 2011
// Copyright 2011, All rights reserved

package bsm;

message LorentzVector
{
    required double e = 1;
    required double px = 2;
    required double py = 3;
    required double pz = 4;
}

message Vector
{
    required double x = 1;
    required double y = 2;
    required double z = 3;
}

message PhysicsObject
{
    required LorentzVector p4 = 1;
    required Vector vertex = 2;
}
